sequencer/src/api/endpoints.rs

```diff
@@ -65,9 +66,35 @@ where
                 resource: height.to_string(),
             })?;
 
-            let proof = block.payload().get_namespace_proof(namespace.into());
+            // TODO fake VidScheme construction
+            let vid = crate::block::payload::test_vid_factory();
+            use hotshot::traits::BlockPayload;
+            let disperse_data = vid
+                .disperse(block.payload().encode().unwrap().collect::<Vec<u8>>())
```

Comment:
Can you create an issue so we don't forget: we should replace this with fetching the VID common data for this block (once the query service supports it) and using that to construct the VID scheme (once HotShot supports it), thus avoiding the need to disperse here

Reply:
#1093